application/account-management/Api/Api.csproj

```diff
@@ -6,6 +6,7 @@
         <RootNamespace>PlatformPlatform.AccountManagement.Api</RootNamespace>
         <Nullable>enable</Nullable>
         <ImplicitUsings>enable</ImplicitUsings>
+        <ContainerFamily>jammy-chiseled</ContainerFamily>
```

Comment:
I believe we should/could also add `<PublishTrimmed>true</PublishTrimmed>` to get the trimmed and chiseled version. See https://learn.microsoft.com/en-us/dotnet/core/deploying/trimming/trim-self-contained

![image](https://github.com/platformplatform/PlatformPlatform/assets/1560381/318cfed4-eb7b-4724-963a-aa8ee656780e)

Reply:
Unfortunately the trimmed version will give us an exception.